packages/forms-web-app/src/controllers/selected-appeal/selected-appeal.js

```diff
@@ -54,15 +54,40 @@ exports.get = async (req, res) => {
 
 	const viewContext = {
 		appeal: {
-			appealNumber: appealNumber,
+			appealNumber,
 			headlineData,
-			// placeholder sections info
 			sections: userSections[userType].map((section) => ({
 				...section,
 				links: section.links.filter(({ condition }) => condition(caseData))
-			}))
+			})),
+			decision: formatDecision({ ...caseData, caseDecisionOutcome: 'invalid' })
 		}
 	};
 
 	res.render(VIEW.SELECTED_APPEAL.APPEAL, viewContext);
 };
+
+/**
+ * @param {import('appeals-service-api').Api.AppealCase} caseData
+ * @returns {{ color: string, label: string } | null}
+ */
+const formatDecision = ({ caseDecisionOutcome }) => {
```

Comment:
There's no ticket for IPs to see outcomes so I figured this would be ok to live here.

Reply:
We already have `mapDecisionColour` in business rules utils, that we should use. There ae also constants for the values there.

Reply:
Sweet, I've moved this func into that file. I had a look at other places where the `mapDecisionColour` was being used and it didn't seem like they did any formatting on the outcome text but maybe it'll be useful